plugins/okta/admin/src/utils/getTrad.js

```diff
@@ -0,0 +1,5 @@
+import pluginId from '../pluginId';
+
+const getTrad = id => `${pluginId}.${id}`;
```

Comment:
Can we rename it to getL10n ? since `trad` isn't a good name for translations. 
Also update it in file name and other files where we use "Trad" as name